charts/lagoon-core/templates/broker.servicemonitor.yaml

```
@@ -0,0 +1,17 @@
+{{- if .Values.broker.serviceMonitor.enabled -}}
+apiVersion: v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "lagoon-core.broker.fullname" . }}
+  labels:
+    {{- include "lagoon-core.broker.labels" . | nindent 4 }}
+spec:
+  endpoints:
+    - port: metrics
+  namespaceSelector:
+    matchNames:
+      - lagoon-core
```

I think this should be something like:
```suggestion
      - {{ .Release.Namespace }}
```